tests/health_checks/NDv4/run-health-checks.sh

```diff
@@ -7,12 +7,44 @@
 #Verify if an argument was passed to the script. Test if the argument is
 #a directory that contains all the expected health checks. If no arguments
 #are passed use the default directory.
+declare -a files=("check-hardware.sh" "dcgmi-check.sh" "ecc-errors.sh"\
+	   "test-bandwidth.sh" "test-ib.sh" "test-nccl.sh"\
+	   "test-MaxPayLoad.sh")
 if [ "$#" -eq 0 ]; then
-	TEST_DIR="/opt/azurehpc/test/health_checks/NDv4/"
+	pass=1
+	DIR=$(cd `dirname $0` && pwd)
+	for file in "${files[@]}"; do
+		if test -f "$DIR/$file"; then
+			true
+		else
+			pass=0
+		fi
```

Comment:
We only need to check non-existent case, then set pass to 0, right?

Reply:
yes, that's right. I will change this.